### PR TITLE
fix: public runtimeconfig types

### PR DESCRIPTION
**Pull request by Hebilicious**
Attempt to fix the type error that happens due to nuxt 3.6

### CODE REVIEW

File: packages/form-actions-nuxt/src/types.d.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+declare module "nuxt/schema" {
+    interface PublicRuntimeConfig {
+      __serverLoaders__: string[]
+    }
+}
+
+export { }
```
Comment (danielroe) [inline packages/form-actions-nuxt/src/types.d.ts:7]: this isn't being distributed with your module. You can add it to `runtime` directory and then add it to the end user's project with `prepare:types` hook. Or add a type template to do this. You might even be able just to move these lines into `module.ts`, but I haven't confirmed if that's enough.

Comment (Hebilicious) [reply packages/form-actions-nuxt/src/types.d.ts:7]: I initially had it in module.ts and that didn't work. It's a little annoying that this needs to be manually included as part of the module runtime T_T
Maybe we can have a convention that `nuxt module bundler` will detect, like `nuxt-types.d.ts` in `src` ?

**Comment by Hebilicious** [comment]
@danielroe Unfortunately your suggestion doesn't work. It looks like `nuxt-build-module` requires additional configuration now.

File: packages/form-actions-nuxt/src/types.d.ts (hunk 1 above)
Comment (danielroe) [reply packages/form-actions-nuxt/src/types.d.ts]: Oh wait, this is only intended within the module code, ie not for end users? You can probably move the types.d.ts out of src then. It shouldn't be in the build in that case

**Pull request merged**